### DISCUSSION TITLE
How to upload files to an existing deployment?

### DISCUSSION THREAD

**Discussion by Alecyrus** [vercel/vercel#4638] [Help]
I have created a deployment (A) via `Vercel Client`, and I have no idea about how to upload updated files to  deployment (A) in another computer.  `Vercel Client` only support `createDeployment `.

**Comment by paulogdm** [comment]
Deployments are immutable. You can't "update files" in an existing deployment and you should create a new deployment. However, you can re-use files and only upload the ones that changed.

Does that answer your question?

**Reply by Alecyrus** [reply]
I got it, and is there a way to create a new development for an existing project? Actually, I just want to find a programmatic approach to manage multiple projects, but some concepts are confused. I don't figure out about how to control and manage projects. I create a project but there is no way to  create a development for this project.

**Reply by paulogdm** [reply]
It works like this:
![untitled (7)](https://user-images.githubusercontent.com/7690649/84572075-f9ab1200-ad6d-11ea-84d0-0c3ff158bd00.png)

**Comment by Alecyrus** [comment]
After creating an empty project by calling `Create a Project` API,  I can't attach a `git repo` to or create a development for this empty project

**Comment by Alecyrus** [comment]
I notice that Vervel CLI could configure `Project Linking`, but the feature is not provided by `Vercel API`

**Reply by paulogdm** [reply]
This is not an API feature, but simply a "CLI" utility.
